packages/client/src/components/image/Image.tsx

```diff
@@ -0,0 +1,15 @@
+import { TImage } from '../../../types/image';
```

Comment:
I rather like to see all of the global types import comes from one source `import { TImage } from '../../../types`, and do not want to have multiple imports for a particular type e.g. `import { TImage } from '../../../types/image`